### PR TITLE
added a test-only function to return SignerCapabilityOfferProofChallengeV2

### PR DESCRIPTION
**Pull request by dhruvja**
### Description
This PR is regarding this issue #6109

### CODE REVIEW

**Review by alinush — APPROVED** [review]
LGTM! @0xchloe?

File: aptos-move/framework/aptos-framework/sources/account.move [hunk 1]
```diff
@@ -822,6 +822,16 @@ module aptos_framework::account {
         revoke_signer_capability(&alice, charlie_addr);
     }
 
+    #[test_only]
+    public fun get_signer_capability_offer_proof_challenge_V2(source_address: address, recipient_address: address) :SignerCapabilityOfferProofChallengeV2 {
```
Comment (alinush) [inline aptos-move/framework/aptos-framework/sources/account.move:826]: change `' :'` to `': '`

Comment (dhruvja) [reply aptos-move/framework/aptos-framework/sources/account.move]: Done

**Review by 0xchloe — APPROVED** [review]
lgtm

File: aptos-move/framework/aptos-framework/sources/account.move [hunk 2]
```diff
@@ -822,6 +822,16 @@ module aptos_framework::account {
         revoke_signer_capability(&alice, charlie_addr);
     }
 
+    #[test_only]
+    public fun get_signer_capability_offer_proof_challenge_V2(source_address: address, recipient_address: address): SignerCapabilityOfferProofChallengeV2 acquires Account {
+       let challenge = SignerCapabilityOfferProofChallengeV2 {
+            sequence_number: get_sequence_number(source_address),
+            source_address: source_address,
+            recipient_address: recipient_address,
+        };
```
Comment (gregnazario) [inline aptos-move/framework/aptos-framework/sources/account.move:831]: Please remove the blank space at the end of this line, and it should pass the linter then.

Comment (dhruvja) [reply aptos-move/framework/aptos-framework/sources/account.move]: done

**Review by davidiw — APPROVED** [review]

**Comment by davidiw** [comment]
Please run cargo xclippy and fix the recommendations.

**Comment by davidiw** [comment]
well I broke this, so you'll need a new pr

**Pull request closed without merging**